src/lib.rs

```diff
@@ -21,6 +21,24 @@ pub enum Interface {
     Any,
 }
 
+pub enum FlowControl {
+    Disable,
+    RtsCts,
+    DtrDsr,
+    XonXoff
+}
+
+impl Into<i32> for FlowControl {
```

Comment:
I'd prefer to have this as a private method for now. The specific codes used by FTDI for this should not be part of our API.